app/controllers/courses_controller.rb

```diff
@@ -157,6 +157,17 @@ def self.registered(app)
             sorting = params_sorting_array 'course_id'
             query   = params_search_query  @special_params
 
+            # use regex instead of just 'gen_ed = DSHS'
+            # now gen ed code may be in the middle of a string (DSHS or DVUP, DSHS if taken with..., etc)
+            if query['gen_ed']
+              new_gened_query = {
+                '$regex': ".*#{query['gen_ed']}.*",
```

Comment:
why do you have the `.*` before and after the gen_ed? wouldn't just a regex search for gen_ed work?